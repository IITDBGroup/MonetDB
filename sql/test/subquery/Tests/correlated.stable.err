stderr of test 'correlated` in directory 'sql/test/subquery` itself:


# 20:30:33 >  
# 20:30:33 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=37437" "--set" "mapi_usock=/var/tmp/mtest-8372/.s.monetdb.37437" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_subquery" "--set" "embedded_c=true"
# 20:30:33 >  

# builtin opt 	gdk_dbpath = /home/niels/scratch/monetdb/Linux-x86_64/var/monetdb5/dbfarm/demo
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 37437
# cmdline opt 	mapi_usock = /var/tmp/mtest-8372/.s.monetdb.37437
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /home/niels/scratch/monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_subquery
# cmdline opt 	embedded_c = true
#client2:!ERROR:SQLException:assert:M0M29!21000!zero_or_one: cardinality violation, scalar expression expected
#client2:!ERROR:ParseException:SQLparser:42000!syntax error, unexpected LIMIT, expecting UNION or EXCEPT or INTERSECT or ')' in: "select i, (select 42+i1.i from integers limit"
#client2:!ERROR:ParseException:SQLparser:42000!syntax error, unexpected LIMIT, expecting UNION or EXCEPT or INTERSECT or ')' in: "select i, (select 42+i1.i from integers limit"
#client2:!ERROR:ParseException:SQLparser:42000!syntax error, unexpected LIMIT, expecting UNION or EXCEPT or INTERSECT or ')' in: "select i, (select i+i1.i from integers limit"
#client2:!ERROR:ParseException:SQLparser:42000!Cannot use non GROUP BY column 'i' in query results without an aggregate function
#client2:!ERROR:ParseException:SQLparser:42000!Cannot use non GROUP BY column in query results without an aggregate function
#client2:!ERROR:ParseException:SQLparser:42000!Subquery result missing
#client2:!ERROR:ParseException:SQLparser:42000!SELECT: subquery result missing
#main thread:!ERROR:MALException:client.quit:Server stopped

# 20:30:33 >  
# 20:30:33 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-8372" "--port=37437"
# 20:30:33 >  

MAPI  = (monetdb) /var/tmp/mtest-8372/.s.monetdb.37437
QUERY = SELECT i, (SELECT 42+i1.i FROM integers) AS j FROM integers i1 ORDER BY i;
ERROR = !21000!zero_or_one: cardinality violation, scalar expression expected
CODE  = M0M29
MAPI  = (monetdb) /var/tmp/mtest-8372/.s.monetdb.37437
QUERY = SELECT i, (SELECT 42+i1.i FROM integers LIMIT 1) AS j FROM integers i1 ORDER BY i;
ERROR = !syntax error, unexpected LIMIT, expecting UNION or EXCEPT or INTERSECT or ')' in: "select i, (select 42+i1.i from integers limit"
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-8372/.s.monetdb.37437
QUERY = SELECT i, (SELECT 42+i1.i FROM integers LIMIT 0) AS j FROM integers i1 ORDER BY i;
ERROR = !syntax error, unexpected LIMIT, expecting UNION or EXCEPT or INTERSECT or ')' in: "select i, (select 42+i1.i from integers limit"
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-8372/.s.monetdb.37437
QUERY = SELECT i, (SELECT i+i1.i FROM integers LIMIT 1 OFFSET 1) AS j FROM integers i1 ORDER BY i; -- errror
ERROR = !syntax error, unexpected LIMIT, expecting UNION or EXCEPT or INTERSECT or ')' in: "select i, (select i+i1.i from integers limit"
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-8372/.s.monetdb.37437
QUERY = SELECT i, (SELECT SUM(i1.i)) FROM integers i1;
ERROR = !Cannot use non GROUP BY column 'i' in query results without an aggregate function
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-8372/.s.monetdb.37437
QUERY = SELECT i+1, (SELECT SUM(i1.i)) FROM integers i1;
ERROR = !Cannot use non GROUP BY column in query results without an aggregate function
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-8372/.s.monetdb.37437
QUERY = SELECT SUM(i), (SELECT SUM(i) FROM integers WHERE i>SUM(i1.i)) FROM integers i1; -- 6, NULL
ERROR = !Subquery result missing
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-8372/.s.monetdb.37437
QUERY = SELECT SUM((SELECT SUM(i))) FROM integers; -- error
ERROR = !SUM: aggregate functions cannot be nested
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-11357/.s.monetdb.35326
QUERY = SELECT (SELECT SUM(i + i1.i), 1 FROM integers) FROM integers i1; --error, the subquery should output only one column
ERROR = !SELECT: subquery must return only one column
CODE  = 42000

# 20:30:33 >  
# 20:30:33 >  "Done."
# 20:30:33 >  

