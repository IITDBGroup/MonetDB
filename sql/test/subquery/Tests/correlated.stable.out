stdout of test 'correlated` in directory 'sql/test/subquery` itself:


# 20:30:33 >  
# 20:30:33 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=37437" "--set" "mapi_usock=/var/tmp/mtest-8372/.s.monetdb.37437" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_subquery" "--set" "embedded_c=true"
# 20:30:33 >  

# MonetDB 5 server v11.34.0 (hg id: 9b9d1f1578b1+)
# This is an unreleased version
# Serving database 'mTests_sql_test_subquery', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 15.385 GiB available main-memory.
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2019 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://xps13:37437/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-8372/.s.monetdb.37437
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

# 20:30:33 >  
# 20:30:33 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-8372" "--port=37437"
# 20:30:33 >  

#CREATE TABLE integers(i INTEGER);
#INSERT INTO integers VALUES (1), (2), (3), (NULL);
[ 4	]
#SELECT i, (SELECT 42+i1.i) AS j FROM integers i1 ORDER BY i;
% sys.i1,	sys.L3 # table_name
% i,	j # name
% int,	bigint # type
% 1,	2 # length
[ NULL,	NULL	]
[ 1,	43	]
[ 2,	44	]
[ 3,	45	]
#SELECT i FROM integers i1 ORDER BY (SELECT 100-i1.i); -- NULL, 3, 2, 1
% sys.i1 # table_name
% i # name
% int # type
% 1 # length
[ NULL	]
[ 3	]
[ 2	]
[ 1	]
#SELECT i, (SELECT i FROM integers WHERE 1=0 AND i1.i=i) AS j FROM integers i1 ORDER BY i;
% sys.i1,	.L5 # table_name
% i,	j # name
% int,	int # type
% 1,	1 # length
[ NULL,	NULL	]
[ 1,	NULL	]
[ 2,	NULL	]
[ 3,	NULL	]
#SELECT i, EXISTS(SELECT i FROM integers WHERE 1=0 AND i1.i=i) AS j FROM integers i1 ORDER BY i;
% sys.i1,	.L5 # table_name
% i,	j # name
% int,	boolean # type
% 1,	5 # length
[ NULL,	NULL	]
[ 1,	NULL	]
[ 2,	NULL	]
[ 3,	NULL	]
#SELECT i, i=ANY(SELECT i FROM integers WHERE 1=0 AND i1.i=i) AS j FROM integers i1 ORDER BY i;
% .i1,	.L4 # table_name
% i,	j # name
% int,	boolean # type
% 1,	5 # length
[ NULL,	NULL	]
[ 1,	NULL	]
[ 2,	NULL	]
[ 3,	NULL	]
#SELECT i, (SELECT 42 WHERE i1.i>2) AS j FROM integers i1 ORDER BY i;
% .i1,	.L5 # table_name
% i,	j # name
% int,	tinyint # type
% 1,	2 # length
[ NULL,	NULL	]
[ 1,	NULL	]
[ 2,	NULL	]
[ 3,	42	]
#SELECT i, (SELECT 42 WHERE i1.i IS NULL) AS j FROM integers i1 ORDER BY i;
% .i1,	.L5 # table_name
% i,	j # name
% int,	tinyint # type
% 1,	2 # length
[ NULL,	42	]
[ 1,	NULL	]
[ 2,	NULL	]
[ 3,	NULL	]
#SELECT i, (SELECT i+i1.i FROM integers WHERE i=1) AS j FROM integers i1 ORDER BY i;
% sys.i1,	.L7 # table_name
% i,	j # name
% int,	bigint # type
% 1,	1 # length
[ NULL,	NULL	]
[ 1,	2	]
[ 2,	3	]
[ 3,	4	]
#SELECT i, (SELECT i FROM integers WHERE i=i1.i) AS j FROM integers i1 ORDER BY i;
% sys.i1,	.L5 # table_name
% i,	j # name
% int,	int # type
% 1,	1 # length
[ NULL,	NULL	]
[ 1,	1	]
[ 2,	2	]
[ 3,	3	]
#SELECT i, (SELECT i+1 FROM integers WHERE i=i1.i) AS j FROM integers i1 ORDER BY i;
% sys.i1,	.L6 # table_name
% i,	j # name
% int,	bigint # type
% 1,	1 # length
[ NULL,	NULL	]
[ 1,	2	]
[ 2,	3	]
[ 3,	4	]
#SELECT i, (SELECT 42 FROM integers WHERE i=i1.i) AS j FROM integers i1 ORDER BY i;
% sys.i1,	.L6 # table_name
% i,	j # name
% int,	tinyint # type
% 1,	2 # length
[ NULL,	NULL	]
[ 1,	42	]
[ 2,	42	]
[ 3,	42	]
#SELECT i, (SELECT MIN(i)+i1.i FROM integers) FROM integers i1 ORDER BY i;
% .i1,	.L11 # table_name
% i,	L11 # name
% int,	bigint # type
% 1,	1 # length
[ NULL,	NULL	]
[ 1,	2	]
[ 2,	3	]
[ 3,	4	]
#SELECT i, (SELECT MIN(i+2*i1.i) FROM integers) FROM integers i1 ORDER BY i;
% sys.i1,	.L7 # table_name
% i,	L7 # name
% int,	bigint # type
% 1,	1 # length
[ NULL,	NULL	]
[ 1,	3	]
[ 2,	5	]
[ 3,	7	]
#SELECT i, CAST(SUM(i) AS BIGINT), CAST((SELECT SUM(i)+SUM(i1.i) FROM integers) AS BIGINT) FROM integers i1 GROUP BY i ORDER BY i;
% sys.i1,	sys.L4,	.L16 # table_name
% i,	L4,	L16 # name
% int,	bigint,	bigint # type
% 1,	1,	2 # length
[ NULL,	NULL,	NULL	]
[ 1,	1,	10	]
[ 2,	2,	14	]
[ 3,	3,	18	]
#SELECT i, CAST(SUM(i) AS BIGINT), CAST((SELECT SUM(i)+COUNT(i1.i) FROM integers) AS BIGINT) FROM integers i1 GROUP BY i ORDER BY i;
% sys.i1,	sys.L4,	.L16 # table_name
% i,	L4,	L16 # name
% int,	bigint,	bigint # type
% 1,	1,	2 # length
[ NULL,	NULL,	6	]
[ 1,	1,	10	]
[ 2,	2,	10	]
[ 3,	3,	10	]
#SELECT i, (SELECT COUNT(*) FROM integers i2 WHERE i2.i>i1.i) FROM integers i1 ORDER BY i;
% sys.i1,	.L5 # table_name
% i,	L5 # name
% int,	bigint # type
% 1,	1 # length
[ NULL,	0	]
[ 1,	2	]
[ 2,	1	]
[ 3,	0	]
#SELECT i, (SELECT MIN(i+2*i1.i) FROM integers) FROM integers i1 ORDER BY i;
% sys.i1,	.L7 # table_name
% i,	L7 # name
% int,	bigint # type
% 1,	1 # length
[ NULL,	NULL	]
[ 1,	3	]
[ 2,	5	]
[ 3,	7	]
#SELECT CAST((SELECT SUM(i1.i)) AS BIGINT) FROM integers i1; -- 6
% sys.L4 # table_name
% L4 # name
% bigint # type
% 1 # length
[ 6	]
#SELECT MIN(i), CAST((SELECT SUM(i1.i)) AS BIGINT) FROM integers i1; -- 1, 6
% sys.L3,	sys.L6 # table_name
% L3,	L6 # name
% int,	bigint # type
% 1,	1 # length
[ 1,	6	]
#SELECT MIN(i), CAST((SELECT SUM(i1.i)) AS BIGINT) FROM integers i1; -- 1, 6
% sys.L3,	sys.L6 # table_name
% L3,	L6 # name
% int,	bigint # type
% 1,	1 # length
[ 1,	6	]
#SELECT CAST((SELECT SUM(i1.i)) AS BIGINT), CAST((SELECT SUM(i1.i)) AS BIGINT) FROM integers i1; -- 6, 6
% sys.L4,	sys.L7 # table_name
% L4,	L7 # name
% bigint,	bigint # type
% 1,	1 # length
[ 6,	6	]
#SELECT CAST(SUM(i) AS BIGINT), CAST(SUM((SELECT i FROM integers WHERE i=i1.i)) AS BIGINT) FROM integers i1; -- 6, 6
% sys.L4,	.L11 # table_name
% L4,	L11 # name
% bigint,	bigint # type
% 1,	1 # length
[ 6,	6	]
#SELECT i, (SELECT MIN(i) FROM integers WHERE i>i1.i) FROM integers i1 ORDER BY i;
% sys.i1,	.L5 # table_name
% i,	L5 # name
% int,	int # type
% 1,	1 # length
[ NULL,	NULL	]
[ 1,	2	]
[ 2,	3	]
[ 3,	NULL	]
#SELECT i, (SELECT MIN(i)+i1.i FROM integers WHERE i>i1.i) FROM integers i1 ORDER BY i;
% sys.i1,	.L6 # table_name
% i,	L6 # name
% int,	bigint # type
% 1,	1 # length
[ NULL,	NULL	]
[ 1,	3	]
[ 2,	5	]
[ 3,	NULL	]

# 20:30:33 >  
# 20:30:33 >  "Done."
# 20:30:33 >  

